.gitignore

```diff
@@ -30,3 +30,10 @@
 /redexdump
 /stamp-h1
 /test-driver
+
+
+redex-src-strings-map.txt
+pyredex/__pycache__/__init__.cpython-36.pyc
```

Comment:
nit-pick: why not ignore all `.pyc` files?

Reply:
I wasn't sure if any of them would actually be needed in the future.